src/core_codemods/docs/pixee_python_django-json-response-type.md

```diff
@@ -0,0 +1,11 @@
+For a `HttpReponse` in Django, if the `content_type` is not set, it default to `text/html`. If the JSON contains user supplied input without being sanitized first, a malicious user may supply HTTP code which leaves the application vulnerable to cross-site scripting (XSS). This transformation restricts the response type to `application/json` to avoid this vulnerability. Our changes look something like this:
```

Comment:
Just a suggestion.
```suggestion
The default `content_type` for `HttpResponse` in Django is `'text/html'`. This is true even when the response contains JSON data.

If the JSON contains (unsanitized) user-supplied input, a malicious user may supply HTML code which leaves the application vulnerable to cross-site scripting (XSS). 

This fix explicitly sets the response type to `application/json` when the response body is JSON data to avoid this vulnerability. Our changes look something like this:
```